packages/xo-web/src/xo-app/jobs/new/index.js

```diff
@@ -38,6 +39,43 @@ import {
 
 const JOB_KEY = 'genericTask'
 
+const COLUMNS = [
+  {
+    itemRenderer: (job, isJobUserMissing) => {
+      const { id } = job
+
+      return (
+        <div>
+          {job.name} <span className='text-muted'>({id.slice(4, 8)})</span>
+          {!isJobUserMissing[id] && (
+            <Tooltip content={_('jobUserNotFound')}>
+              <Icon className='ml-1' icon='error' />
+            </Tooltip>
+          )}
+        </div>
+      )
+    },
+    name: _('jobName'),
+    sortCriteria: 'name',
+  },
+  {
+    itemRenderer: job => job.method,
+    name: _('jobAction'),
+    sortCriteria: 'method',
+  },
+]
+
+const ACTIONS = [
+  {
+    handler: deleteJobs,
+    handlerParam: selectedJob => selectedJob.id,
```

Comment:
This looks unnecessary, why not simply define your handler like this:

```js
handler: jobs => deleteJobs(jobs.map(job => job.id))
```

Or it could simply be handled directly in `deleteJobs`: 

```js
const ids = jobs.map(resolveId)
```